xbmc/win32/Win32DelayedDllLoad.cpp

```diff
@@ -70,5 +70,5 @@ FARPROC WINAPI delayHookFailureFunc (unsigned dliNotify, PDelayLoadInfo pdli)
 }
 
 // assign hook functions
-PfnDliHook __pfnDliNotifyHook2 = delayHookNotifyFunc;
-PfnDliHook __pfnDliFailureHook2 = delayHookFailureFunc;
```

Comment:
@Paxxi 
> oh I missed this, for the love of god why did this get merged with these lines commented out? these are critical for dll loading so this should never have been merged

I really wonder why it breaks my build on VS 2015 Update 3. Also all my add-ons worked without that lines. Somewhere in the VS header files these lines are already defined.

Do you need the corresponding file?